cardano-node/src/Cardano/Tracing/Util.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE EmptyCase #-}
```

Comment:
I suspect a lot of these pragmas are unnecessary e.g `EmptyCase`

Reply:
Fixed.